app/models/volunteer.rb

```diff
@@ -2,5 +2,13 @@
 
 class Volunteer < ApplicationRecord
   belongs_to :user
-  # has_many :repairs, through: :volunteer_repair
+  validates  :first_name, :last_name, :mobile_phone, :gender, :city, :state,
+             :employer, :position, :availability, :skill, presence: true
+  validate   :invalid_birthdate
+
+  def invalid_birthdate
+    if birthdate.present? && birthdate > Date.today
+      errors.add(:birthdate, "can't be now or in future!")
```

Comment:
Small grammar change: can't be now or in the future.  Don't need the exclamation point.